train.py

```diff
@@ -183,15 +190,7 @@ def train(config):
         if not args.exp_version
         else args.exp_version
     )
-    out_path = Path(config.result_path) / config.exp_name / config.exp_version
-
-    if config.get("wandb", False):
-        run = wandb.init()
 
+    out_path = Path(config.result_path) / config.exp_name / config.exp_version
```

Comment:
Extract path format convenction to function (it is used twice) or move this line and line below to `train` function body (so we will have path format definition just once).